src/Sarif.Driver/DriverExtensionMethods.cs

```diff
@@ -169,6 +169,10 @@ public static bool ValidateOutputOptions(this AnalyzeOptionsBase options, IAnaly
             // baseline process now depends on output file
             valid &= !(string.IsNullOrWhiteSpace(options.OutputFilePath) && !string.IsNullOrWhiteSpace(options.BaselineSarifFile));
 
+            // post SARIF process now depends on output file
```

Comment:
>post SARIF process now depends on output file [](http://example.com/codeflow?start=15&length=45)

? What does this comment mean? 

can you delete it? and Delete the comment on line 170?